src/libs/ReportUtils.js

```diff
@@ -1617,9 +1617,10 @@ function getTransactionDetails(transaction, createdDateFormat = CONST.DATE.FNS_F
  *    - or the user is an admin on the policy the expense report is tied to
  *
  * @param {Object} reportAction
+ * @param {String} fieldToEdit
  * @returns {Boolean}
  */
-function canEditMoneyRequest(reportAction) {
+function canEditMoneyRequest(reportAction, fieldToEdit) {
```

Comment:
Isn't fieldToEdit optional? I see the function being called in more places so we probably need a default

Reply:
Just updated. Added the default value to ```fieldToEdit``` param. Thanks